commands/substr.md

```diff
@@ -0,0 +1,22 @@
+Returns the substring of the string value stored at `key`, determined by the
```

Comment:
Missing depreciation notice?

Reply:
Or maybe it's suppose to be auto generated, and we need to remove it (and other Metadata like history and complexity) from all other files?

Reply:
Exactly. I can actually make the change to redis-io to use the new deprecation metadata now, but I want to clear it from the files first. Perhaps we'll leave a deprecation notice in substr just for now?

Reply:
if you wish to do it in steps, ok..
but let's not forget.